features/support/remote_helpers.rb

```diff
@@ -12,13 +12,13 @@ def base_url domain
 
 
 # Returns the remote URL for a new pull request for the given domain and branch
-def pull_request_url domain, branch_name, repo
+def pull_request_url domain, branch, parent_branch, repo
   case domain
   when 'Bitbucket'
     sha = recent_commit_shas(1).join('')[0, 12]
-    "https://bitbucket.org/#{repo}/pull-request/new?source=#{CGI.escape repo}%3A#{sha}%3A#{branch_name}"
+    "https://bitbucket.org/#{repo}/pull-request/new?source=#{CGI.escape repo}%3A#{sha}%3A#{branch}"
```

Comment:
Agreed. I don't have access to a BitBucket-based repo to test this right now, so I didn't do this for now. One of you guys has to add this later.